tools/building.py

```diff
@@ -79,6 +79,7 @@ def get_building_env():
   env['PKG_CONFIG_PATH'] = os.environ.get('EM_PKG_CONFIG_PATH', '')
   env['EMSCRIPTEN'] = path_from_root()
   env['PATH'] = cache.get_sysroot_dir('bin') + os.pathsep + env['PATH']
+  env['ACLOCAL_PATH'] = cache.get_sysroot_dir('share/aclocal')
```

Comment:
lgtm.. but I wonder if there is a way to specify the sysroot such that aclocal will then derive this?

Reply:
Yeah me too... Haven't found it yet though.

Reply:
From what I hear, /usr is basically hardcoded in tons of places in autotools, hence all the custom Vars.

Reply:
Ah I remember now, I think I came across it back then and we even talked about it I think?

Anyway, apparently there is CONFIG_SITE where you can put a shell script to override anything you want for autotools-based configure files, even `--prefix` so that users wouldn't have to supply one themselves: https://www.gnu.org/software/autoconf/manual/autoconf-2.63/html_node/Site-Defaults.html

Reply:
Ohhh apparently then we could even automatically cache system-specific (aka Emscripten-specific) results in sysroot, so all the `./configure` would run faster.

I'll try to make a PR.